fv3core/utils/global_constants.py

```diff
@@ -35,3 +35,9 @@
 t_sub = 184.0  # min temp for sublimation of cloud ice
 DC_ICE = C_LIQ - C_ICE
 LI0 = HLF - DC_ICE * TICE
+
+# nq is actually given by ncnst - pnats, where those are given in atmosphere.F90 by:
+# ncnst = Atm(mytile)%ncnst
+# pnats = Atm(mytile)%flagstruct%pnats
+# here we hard-coded it because 8 is the only supported value, refactor this later!
+NQ = 8  # state.nq_tot - spec.namelist.dnats
```

Comment:
I don't know how this crept in, but this should be deleted. NQ should be hard-coded in the `fv_dynamics` module instead since it's not a physical constant.

Reply:
It's possible I missed that in a merge conflict. Happy to cut that